packages/ui/origin-ui-core/src/components/Modal/IRecAccountRegisteredModal.tsx

```diff
@@ -1,45 +1,29 @@
 import React from 'react';
-import { useHistory } from 'react-router-dom';
-import { useSelector } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 import { Dialog, DialogTitle, DialogActions, Button, Box, useTheme, Grid } from '@material-ui/core';
-import { getUserOffchain } from '../../features/users';
-import { showNotification, NotificationType } from '../../utils/notifications';
-import { useLinks } from '../../utils/routing';
+import { showNotification, NotificationType } from '../../utils';
 import iconAdded from '../../../assets/icon-org-added.svg';
 
 interface IProps {
     showModal: boolean;
     setShowModal: (showModal: boolean) => void;
-    setShowBlockchainModal?: (showModal: boolean) => void;
+    onClose?: (showModal: boolean) => void;
```

Comment:
This naming is misleading, because now it looks like event handler, but actually it is a setState function passed as prop.